src/Apps/Auction/Components/LotInfo.tsx

```diff
@@ -47,20 +47,20 @@ export const LotInfo: React.FC<Props> = ({ artwork, saleArtwork }) => {
 export const LotInfoFragmentContainer = createFragmentContainer(LotInfo, {
   artwork: graphql`
     fragment LotInfo_artwork on Artwork {
-      _id
+      internalID
       date
       title
       imageUrl
-      artistNames: artist_names
+      artistNames: artistNames
```

Comment:
Would it be complicated to evaluate whether each selection is identical before writing the addition?

Reply:
Hmm, I’m surprised by this, in most places we did check if the new name was different than what was there already. Anyways, it should be easy to write a little codemod to clean this up post-migration 👍